docs/source/complex_numbers.rst

```diff
@@ -0,0 +1,174 @@
+.. _complex_numbers-doc:
+
+Complex Numbers
+===============
+
+Complex Numbers are numbers that can be expressed in the form :math:`a + bi`, where a and b are real numbers,
+and *i* is a solution of the equation :math:`x^2 = −1`. Complex numbers frequently occur in mathematics and
+engineering, especially in signal processing. The aim of introducing Complex Tensors is to provide a more natural
```

Comment:
nit: this sentence describes the current state, but it's going to be out of date soon after you write it (i.e., we'll have complex support and torchaudio won't have hacks.  You can just say something like "Complex Tensors are provided..."

Reply:
nit: it's a little wordier, but "Tensors of complex dtype" is more descriptive/accurate.

Reply:
nit: I believe "workaround" is two words when used as a verb.